app/ExposureNotificationContext.tsx

```diff
@@ -29,18 +37,32 @@ const ExposureNotificationsProvider = ({
     setExposureNotificationAuthorizationStatus,
   ] = useState<ENAuthorizationStatus>(initialStatus);
 
+  const [diagnosisKeys, setDiagnosisKeys] = useState<ENDiagnosisKey[]>(
+    initialKeys,
+  );
+
   const requestExposureNotificationAuthorization = () => {
     const cb = (authorizationStatus: ENAuthorizationStatus) => {
       setExposureNotificationAuthorizationStatus(authorizationStatus);
     };
     ExposureNotifications.requestAuthorization(cb);
   };
 
+  const fetchDiagnosisKeys = () => {
+    const cb = (error: Error, diagnosisKeys: ENDiagnosisKey[]) => {
```

Comment:
@johnschoeman is there a best practice for handling errors within a context? I'd like to be able to display an alert to the user, so somehow I'd like to pass the error to the caller I think. Related question: will this context have the same role now that `Redux` is gradually being worked in?